workflows/intake-catalog.yaml

```diff
@@ -0,0 +1,69 @@
+# Intake Catalog for the workflow in-progress
+plugins:
+  source: [ module: intake_xarray ]
+sources:
+  cmip6_raw:
+    description: Catalog of raw CMIP6 GCM data to be cleaned and prepared
+    metadata:
+      title: Raw CMIP6 GCM data
+    parameters:
+      source_id:
+        description: The data source ID
+        default: "ACCESS-ESM1-5"
+        type: str
+      experiment_id:
+        description: The experiment ID
+        type: str
+        default: historical
+        allowed: [ historical, ssp126, ssp245, ssp370 ]
+      member_id:
+        description: The model member ID
+        default: "r1i1p1f1"
+        type: str
+      variable_id:
+        description: The data variable ID
+        default: tasmax
+        type: str
+        allowed: [ tasmax, tasmin, pr ]
+      grid_label:
+        description: The data grid label
+        default: gn
+        type: str
+      item_version:
+        description: The data version
+        default: "20191115"
+        type: str
+    driver: zarr
+    args:
+      urlpath: "az://raw/{{ source_id }}/{{ experiment_id }}/{{ member_id }}/{{ variable_id }}/{{ grid_label }}/{{ item_version }}.zarr"
+      storage_options:
+        account_name: dc6
+
+  cmip6_clean:
+    description: Catalog of cleaned, parsed, and prepared CMIP6 GCM data for input to workflow
+    metadata:
+      title: Cleaned, parsed, and prepared CMIP6 GCM data
+    parameters:
+      source_id:
+        description: The data source ID
+        default: "ACCESS-ESM1-5"
+        type: str
+      division:
+        description: The division or role of the data
+        type: str
+        default: historical
+        allowed: [ historical, training, ssp126, ssp245, ssp370 ]
```

Comment:
Is the distinction bw historical and training here that historical would be the full time period and training the 1995-2014 time period?

Reply:
Yes.

Or historical is 1980 - 2014 or whatever people decide on. The distinction is a bit fuzzy, but I def need a "training" set that is trimmed to the reanalysis record.